views/server.info.php

```diff
@@ -26,20 +26,26 @@
 $conf_realtime = $this->extconfigs->validate_RealTime($ast_realm);
 $db_Schema = $this->dbinterface->validate();
 $mysql_info = $this->dbinterface->get_db_sysvalues();
-$compatible = $this->srvinterface->get_compatible_sccp();
+$compatible = $this->aminterface->get_compatible_sccp();
```

Comment:
We should choose for either camelCase or SnakeCase for functions/methods but not both. I prefer camelCase, but that does not matter.